Exercise01/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise01
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Func<int, List<int>> ListOfPrimeSmallerX = (x) =>

```

Comment:
Make `ListOfPrimeSmallerX` to become a class function

Reply:
Done